tools/lib/provision.py

```diff
@@ -197,8 +199,11 @@
         *VENV_DEPENDENCIES,
     ]
 elif "debian" in os_families():
+    REFRACTORED_DEPENDECIES = UBUNTU_COMMON_APT_DEPENDENCIES
+    REFRACTORED_DEPENDECIES.remove("libappindicator1")
     SYSTEM_DEPENDENCIES = [
-        *UBUNTU_COMMON_APT_DEPENDENCIES,
+        *REFRACTORED_DEPENDECIES,
+        "libgroonga0",
```

Comment:
libappindicator1 will be removed only in Debian testing (Bullseye). This change will remove the dependency for all Debian versions

Reply:
IIRC, the package will remain deprecated in the stable release as well and maybe in the future releases as well. I am not quite sure if this package is needed, because this part of the code runs for `bionic` as well since `ubuntu` belongs to `debian` family, but the puppeteer code tests run successfully.

Reply:
> deprecated in the stable release

[Nope](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=895037#24)
> in the future releases

that's what we have to fix *temporarily*. Once the [migration](https://wiki.debian.org/Ayatana/IndicatorsTransition) is done, we can replace `libappindicator1` with `libayatana-appindicator`